Basic setup

NoelLincoln opened the pull request:
Feat:
Create index.html and add h1 tag inside header tag
Create style.css and add background color to header tag using css class selector
Create linters.yml file to check for errors in the code
Add node_modules directory to .gitignore
Create test.md file and add it to .gitignore

NoelLincoln committed:
Add linter configuration to gitignore
Create html file with h1 tag in a header section

NoelLincoln committed:
Add a class attribute to the header tag
Use CSS class selector to change background of header

NoelLincoln committed:
Create a test.md file and add it to gitignore.

NoelLincoln committed:
Remove linters.yml file from .gitignore

ademibowale reviewed (changes requested):
## STATUS: Changes Required :recycle:

### Hi @DevaWinner , 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media.tenor.com/isiLRnxz3zwAAAAM/cat-driving-serious.gif" width="500px" height="300px">

### To Highlight 🥇

✅ Descriptive Pull Request title and summary. ✔️
✅ Good use of Github flow ✔️
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.

### Changes Required

- [ ]  _Please check the comments under the review._
- [ ] Good job using the Microverse template for your README. However, I noticed that some sections are missing from your content table. Please ensure that you include all the main sections in your README as it is a requirement of the project to follow the [Microverse template](https://github.com/microverseinc/readme-template/blob/master/README.md). kindly modify it to fit your custome,The only section that is optional to include is the FAQ section. Also, if your project doesn't require deployment, you can remove the Live Demo section. 🎯
If you don't have a logo you can remove it too.

- [ ] The only optional section is FAQ.
You should delete the live demo section if the you decides not to deploy the application. If project requirements say students should deploy, they should put the link.
Make sure the you have added the section Future Features, including 1-3 potential features to be added.


you should remove the section description, i.e. "> Describe the tech stack and include only the relevant sections that apply to your project." should be removed and filled in with your custom content.
remove template logo. Optionally, the students can replace with their own logo if they have one.you submit for the new review.

Kindly follow the guildline above to modify your readme step by step before you submit for the new review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @Adebowale ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

NoelLincoln commented:
After including the main sections, can I leave some sub-sections blank or do I remove them because it does not relate to my project? For example, under getting started where there's no setup needed, no install, no usage etc.

ademibowale commented:
> After including the main sections, can I leave some sub-sections blank or do I remove them because it does not relate to my project? For example, under getting started where there's no setup needed, no install, no usage etc.



> After including the main sections, can I leave some sub-sections blank or do I remove them because it does not relate to my project? For example, under getting started where there's no setup needed, no install, no usage etc.


On narate how your project can be clone with yoir repo link ,and the usage , only section that is optional to include is the FAQ section.

NoelLincoln commented:
Okay

NoelLincoln committed:
Fix on README file:
    Include missing sections in my content table
    Removed live demo section which was not needed

NoelLincoln committed:
Fix on README:
    Format instructions on usage and how to run project

NoelLincoln commented:
[@Adebowale](https://github.com/Adebowale) 
Should I close this pull request and do a new one after I'm done with the corrections?

NoelLincoln committed:
Fix: Update README file

roshan-bajgain commented on README.md:
````diff
@@ -41,6 +143,5 @@ Hello world program
 
 This project is [MIT](./LICENSE) licensed.
 
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
-
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
+```
````
- [ ] I notice that your MIT license is broken please fix this license so the README file looks professional. Here is the [LInk](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) how you can create license in your README file.

NoelLincoln replied:
@roshan-bajgain 
I've fixed the broken link and used the MIT license.
Thanks

roshan-bajgain reviewed (changes requested):
Hi @NoelLincoln, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

NoelLincoln committed:
Create LICENSE

Generate MIT License from the recommended template

NoelLincoln commented:
I have fixed the broken MIT license from the README

Alusp reviewed (approved):
Hi <STUDENT_MENTION>, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![Congrats](https://media.giphy.com/media/jJQC2puVZpTMO4vUs0/giphy.gif)

## Highlights

- [ ] Descriptive Pull Request title and summary :white_check_mark: 
- [ ] Good use of Github flow. :white_check_mark: 
- [ ] All linters are checked. :white_check_mark: 
- [ ] Project is professionally documented. :white_check_mark: 
- [ ] The right GitHub flow method has been used. :white_check_mark: 
- [ ] Professional pull request title and description. :white_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

NoelLincoln commented:
Thank you @Alusp
